src/System.Private.CoreLib/src/System/RtType.cs

```diff
@@ -4883,8 +4879,210 @@ internal static Type GetTypeFromCLSIDImpl(Guid clsid, String server, bool throwO
 
         #endregion
 
-        #region COM
-        #endregion
+#if FEATURE_COMINTEROP
+        [System.Security.SecuritySafeCritical]
```

Comment:
Delete this attribute. The security transparency attributes were removed for CoreCLR.

Reply:
Done.